src/api/taskstechApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+const authToken = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoyMSwiZXhwIjoxNjI2NTAyMjExfQ.i9E85u3U72x_amkKriiBL5HArbMX5k4l_rFgcXiq09g';
```

Comment:
I guess the auth token would be retrived from local storage once the ```Login``` component is in place. 

but like @giftakari mentioned, that it's best to eliminate the chance to expose security details to the public.

Reply:
Thanks for the feedback, I have created the .env file.